### ISSUE TITLE
Solution File

### ISSUE THREAD

**Issue by YasminAMassoud** [garethjns/Kaggle-EEG#16]
I was able to run the code and obtain the results, however i need to make a new solution file for the data i entered, can you guide me which part i need to work on in the code? @garethjns

**Comment by garethjns** [comment]
Hi @YasminAMassoud,

If I remember correctly, [predict.m](https://github.com/garethjns/Kaggle-EEG/blob/master/predict.m) runs the predictions from both models then create the solution file using [saveSub](https://github.com/garethjns/Kaggle-EEG/blob/master/saveSub.m). It should dump a .csv file to the current working directory.

**Comment by YasminAMassoud** [comment]
@garethjns  I was able to do produce thanks, however can you guide me how can i produce AUC for testing without needing to submit solution file ?

**Comment by garethjns** [comment]
@YasminAMassoud It's possible to calculate the AUC for the training data (see https://github.com/garethjns/Kaggle-EEG/issues/17), however you'll need test set labels to calculate it for the test set. I don't know if these are available or not - if they aren't, it might be worth creating a smaller holdout set from the training data and using that instead.

**Comment by YasminAMassoud** [comment]
Thanks

**Comment by YasminAMassoud** [comment]
-I have got the labels now, can you give any guidelines how to do that ?
-I cant trace where is the part of comparing labels for the training set to get the AUC , can you point where that part is ?
